check_render_logs_opportunity_scan.py

```
@@ -0,0 +1,246 @@
+        print("  1. Use Render CLI: render logs --service <service-name>")
+        print("  2. Check Render dashboard: https://dashboard.render.com")
+        print("  3. Export logs manually and pipe to this script")
+        
+        # Try to read from stdin if available
+        print("\n📥 Reading from stdin (if available)...")
+        try:
+            import select
+            if select.select([sys.stdin], [], [], 0)[0]:
+                logs = sys.stdin.readlines()
+                print(f"Read {len(logs)} lines from stdin")
+        except:
```

Ruff (0.14.4)

Do not use bare `except`

(E722)